src/core/network.js

```diff
@@ -115,20 +133,17 @@ var NetworkManager = (function NetworkManagerClosure() {
         pendingRequest.expectedStatus = 200;
       }
 
-      if (args.onProgressiveData) {
-        // Some legacy browsers might throw an exception.
-        try {
-          xhr.responseType = 'moz-chunked-arraybuffer';
-        } catch(e) {}
-        if (xhr.responseType === 'moz-chunked-arraybuffer') {
-          pendingRequest.onProgressiveData = args.onProgressiveData;
-          pendingRequest.mozChunked = true;
-        } else {
-          xhr.responseType = 'arraybuffer';
-        }
+//#if (GENERIC || FIREFOX || MOZCENTRAL)
+      if (args.onProgressiveData && supportsMozChunked) {
+        xhr.responseType = 'moz-chunked-arraybuffer';
+        pendingRequest.onProgressiveData = args.onProgressiveData;
+        pendingRequest.mozChunked = true;
       } else {
         xhr.responseType = 'arraybuffer';
       }
+//#else
```

Comment:
> but in my previous PR a reviewer suggested to not abuse the preprocessor to eliminate the boolean

Missed that bit -- I'm sorry for confusion. (Now I have to convince @Snuffleupagus to agree with me :) But since we are doing special case for CHROMIUM, I would like to remove a compatibility code for MOZCENTRAL as well, and also to be consistent with stuff we did for fonts in the past. 

I'm thinking creating XMLHttpRequest object for feature test during scripts loading is not cheap process, so my goal to not include this code into production FF code base.

(Side note about `//#if CASE1` `if (check) {` `//#endif` -- I don't think we shall use incomplete if-/while-/try-statements in the preprecessor)